src/main/java/com/MyApp/budgetControl/domain/expense/ExpensesService.java

```diff
@@ -1,28 +1,28 @@
 package com.MyApp.budgetControl.domain.expense;
 
 import lombok.RequiredArgsConstructor;
-import org.springframework.http.HttpStatus;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import org.springframework.web.server.ResponseStatusException;
 
 import java.util.List;
+import java.util.UUID;
 
 @Service
 @RequiredArgsConstructor
 public class ExpensesService {
 
-    private final ExpenseRepository expenseRepository;
+    @Autowired
+    private ExpenseRepository expenseRepository;
```

Comment:
Why You did this change? Its better to have it final and inject through constructor - that makes creating test instances easier